x/sp/genesis.go

```diff
@@ -10,7 +10,10 @@ import (
 // InitGenesis initializes the module's state from a provided genesis state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
 	// this line is used by starport scaffolding # genesis/module/init
-	k.SetParams(ctx, genState.Params)
+	err := k.SetParams(ctx, genState.Params)
```

Comment:
Do you forget to update `gensis.go` in `permission`/`storage` to handle the error?